alloydb-jdbc-connector/pom.xml

```diff
@@ -66,25 +66,21 @@
     <dependency>
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-cloud-alloydb-v1beta</artifactId>
```

Comment:
@ttosta-google as FYI, eventually we'll need to bump this to v1. Right now the Connector-related APIs aren't in v1, but will be soon-ish. When that happens, we'll need to circle back to this and upgrade it.

Reply:
I've created a ticket for it #113

Reply:
Thank you!